.env

```diff
@@ -0,0 +1 @@
+API_KEY = "746ddaff6c6c35161177d073645fcc5d"
```

Comment:
what is this key?

Reply:
this file was not in gitignore and it ended up on github.
fixed